distributions/centos-8/centos-8.json

```diff
@@ -7,14 +7,22 @@
   "x86_64": {
     "image_types": [ "aws", "gcp", "azure", "ami", "vhd" ],
     "repositories": [{
+      "id": "baseos",
       "baseurl": "http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/",
       "rhsm": false
     }, {
+      "id": "appstream",
       "baseurl": "http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/",
       "rhsm": false
     }, {
+      "id": "extras",
       "baseurl": "http://mirror.centos.org/centos/8-stream/extras/x86_64/os/",
       "rhsm": false
+    }, {
+      "id": "google-compute-engine",
+      "baseurl": "https://packages.cloud.google.com/yum/repos/google-compute-engine-el8-x86_64-stable",
+      "rhsm": false,
+      "image_type_tags": ["gcp"]
```

Comment:
We need to define 
```ini
[google-cloud-sdk]
name=Google Cloud SDK
baseurl=https://packages.cloud.google.com/yum/repos/cloud-sdk-el8-x86_64
enabled=1
gpgcheck=1
repo_gpgcheck=0
gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
    https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
```
as well, because the GCE image installs the GCP SDK by default. The reason is that the current RHEL image built by Google contains it and the goal was to replicate the image for now.

Reply:
done! can't believe i missed that many :facepalm: